corehq/apps/linked_domain/views.py

```diff
@@ -377,6 +383,28 @@ def delete_domain_link(self, in_data):
 
     @allow_remote_invocation
     def create_release(self, in_data):
+        error_message = ''
+        try:
+            validate_push(self.request.couch_user, self.domain, in_data['linked_domains'])
+        except NoDownstreamDomainsProvided:
+            error_message = ugettext("No downstream project spaces were selected. Please contact support.")
+        except DomainLinkNotFound:
+            error_message = ugettext(
+                "Links between one or more project spaces do not exist. Please contact support."
+            )
+        except AttemptedPushViolatesConstraints:
+            formatted_domains = ', '.join(in_data['linked_domains'])
+            error_message = ugettext('''
+                The attempted push from {} to {} is disallowed. Please contact support.
```

Comment:
instead of "please contact support" do we want to indicate that this is not allowed under the current plan? I think this messaging makes it seem like a bug rather than something that is intentional?

Reply:
this comment applies to all the "please contact support" messages. I think we want to discourage contacting support if it's either a) an issue with their configuration that they can troubleshoot or b) an issue with their plan disallowing?

I might not be following this properly either...

Reply:
Yeah I'm viewing this state as a bug because in theory the UI should not allow this to happen. How do we typically handle cases where UI should prevent it, but _somehow_ did not?

Reply:
i think this makes sense then. as long as that `notify_exception` is there, then that's about right.